### ISSUE TITLE
Setup `default` rake task to run automated tests, make CI build pass

### ISSUE THREAD

**Issue by nihonjinrxs** [loyno-mathcs/workforwardnola#26]
Travis CI builds are failing because it doesn't know how to run tests on the project.

Someone please fix that.

**Comment by tmloupe** [comment]
Is this similar to what we have to do? https://github.com/rails/rails/blob/master/railties/Rakefile

**Comment by nihonjinrxs** [comment]
Yes, similar.  The particular line that's most important in this case is the [definition of a default task that refers to another defined task in the Rakefile](https://github.com/rails/rails/blob/master/railties/Rakefile#L5).  The default task you define is less important, but it would be good if it ran some tests on the code.  😉🚥
